util/chplenv/compiler_utils.py

```diff
@@ -32,7 +32,7 @@ def CompVersion(version_string):
     are not specified, 0 will be used for their value(s)
     """
     CompVersionT = namedtuple('CompVersion', ['major', 'minor', 'revision', 'build'])
-    match = re.search(u'(\d+)(\.(\d+))?(\.(\d+))?(\.(\d+))?', version_string)
+    match = re.search(r"(\d+)(\.(\d+))?(\.(\d+))?(\.(\d+))?", version_string)
```

Comment:
I think this should be `ur"bla bla"` as long as we support Python 2.7

Reply:
I don't think `ur"bla blah"` is valid syntax in python 3 though:
```
    match = re.search(ur"(\d+)(\.(\d+))?(\.(\d+))?(\.(\d+))?", version_string)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: invalid syntax
```

Reply:
In that case, there is a risk this won't work in Python 2.7, since that would presumably need the `u` part.